packages/server/lib/server-base.ts

```diff
@@ -139,12 +139,12 @@ export abstract class ServerBase<TSocket extends SocketE2E | SocketCt> {
     this._baseUrl = null
     this._fileServer = null
 
-    this._eventBus.on('cross:domain:delaying:html', () => {
+    this._eventBus.on('cross:domain:delaying:html', (request) => {
       this.socket.localBus.once('ready:for:domain', () => {
```

Comment:
just to keep track here that this will need to be updated with #20552 appropriately